homeassistant/components/alarm_control_panel/__init__.py

```diff
@@ -150,6 +155,19 @@ class AlarmControlPanelEntity(Entity, cached_properties=CACHED_PROPERTIES_WITH_A
     _attr_supported_features: AlarmControlPanelEntityFeature = (
         AlarmControlPanelEntityFeature(0)
     )
+    _alarm_control_panel_option_default_code: str = ""
+
+    @final
+    @callback
+    def add_default_code(self, code: str | None) -> str | None:
```

Comment:
I realize this is copied from lock, but `add_default_code` is a really confusing name of this method IMHO. Can we try to come up with something which better describes what it does? Maybe `code_or_default_code`?